crates/optimism/consensus/src/lib.rs

```diff
@@ -111,8 +111,6 @@ impl<ChainSpec: EthChainSpec + OpHardforks, B: Block> Consensus<B>
 
         if self.chain_spec.is_cancun_active_at_timestamp(block.timestamp()) {
             validate_cancun_gas(block)?;
-        } else {
-            return Ok(())
         }
```

Comment:
hmm, if we no longer return here, then 

> // canyon is active, else would have returned already

no longer holds

Reply:
it does because of the Shanghai check above this line. should probably change to only use the op hardfork names to minimise confusion, but thought that could be a follow up pr.

Reply:
ah yeah especially canyon cancun is confusing

Reply:
updated it now though. cancun vs canyon, totally confused me in the first place, hence the bug.

Reply:
me too, :D